utils/vpr_pbtype_to_eblif.py

```diff
@@ -85,15 +123,26 @@ def main(args):
         args.pb_type
     )
 
-    pbtype_name, clocks, inputs, outputs = ports(pbtype_leaf)
+    blif_model, clocks, inputs, outputs, carry = ports(pbtype_leaf)
```

Comment:
Since `blif_model` is overwritten in the next line, I'd use here `_, clocks, inputs, outputs, carry = ...`. Or rename it to `pbtype_name` as the function returns it.

Reply:
Done.